app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for one in people:
+        person = Person(one["name"], one["age"])
+        if ("wife" in one and one["wife"] is not None
+                and one["wife"] in Person.people):
```

Comment:
if ("wife" in one and one["wife"] is not None can be simplified using dict.get()
and one["wife"] in Person.people check is redundant cause you created it a line higher.
Same at line 19-20